specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/IpPool.json

```diff
@@ -92,11 +92,13 @@
                     "200": {
                         "description": "OK",
                         "schema": {
-                            "$ref": "Fabric.json#/definitions/ProvisioningState"
+                            "$ref": "#/definitions/IpPool"
                         }
                     },
                     "202": {
-                        "description": "OK"
+                        "description": "Accepted",
+                        "schema": {
+                            "$ref": "#/definitions/IpPool"
```

Comment:
It's not valid JSON file.

Reply:
What is invalid? I can load it into an online JSON Viewer. I can also load it into a JSON within PowerShell using ConvertTo-JSON

Reply:
Add closing } for "202" response.